### PR TITLE
[zh] sync device-plugins cluster-ip-allocation patch-releases

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -919,6 +921,7 @@ Here are some examples of device plugin implementations:
 * [Intel 设备插件](https://github.com/intel/intel-device-plugins-for-kubernetes)支持
   Intel GPU、FPGA、QAT、VPU、SGX、DSA、DLB 和 IAA 设备
 * [KubeVirt 设备插件](https://github.com/kubevirt/kubernetes-device-plugins)用于硬件辅助的虚拟化
+* [NVIDIA GPU 设备插件](https://github.com/NVIDIA/k8s-device-plugin)NVIDIA 的官方设备插件，用于暴露 NVIDIA GPU 并监控 GPU 健康状态。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:924]:
```suggestion
* [NVIDIA GPU 设备插件](https://github.com/NVIDIA/k8s-device-plugin)NVIDIA 的官方设备插件，用于公布 NVIDIA GPU 和监控 GPU 健康状态。
```

File: content/zh-cn/releases/patch-releases.md [hunk 2]
```diff
@@ -127,6 +129,8 @@ dates for simplicity (every month has it).
 在两个月的维护模式期间，发布管理员可能会删减额外的维护版本以解决：
 
 - CVE（在安全响应委员会的建议下）
+- [漏洞](/zh-cn/docs/reference/issues-security/official-cve-feed/)
+  已分配 CVE ID 的漏洞（在安全响应委员会的建议下）。
```
Comment (mengjiao-liu) [inline content/zh-cn/releases/patch-releases.md:133]: From the [preview](https://deploy-preview-48583--kubernetes-io-main-staging.netlify.app/zh-cn/releases/patch-releases/), it is suggested to change to:
```suggestion
- 已分配 CVE ID 的[漏洞](/zh-cn/docs/reference/issues-security/official-cve-feed/)（在安全响应委员会的建议下）。
```

**Review by mengjiao-liu — COMMENTED** [review]
/lgtm

**Pull request merged**